js/ui/text_box/ui.text_editor.base.js

```diff
@@ -631,8 +635,14 @@ const TextEditorBase = Editor.inherit({
         return element === this._input().get(0);
     },
 
+    _changeValueEventIncludesBlur: function() {
+        const eventOption = this.option(this._getChangeValueEventOptionName());
+
+        return eventOption.includes('blur');
+    },
+
     _preventNestedFocusEvent: function(event) {
-        if(event.isDefaultPrevented()) {
+        if(event.isDefaultPrevented() || this._changeValueEventIncludesBlur()) {
```

Comment:
If you write changed on text_editor level, you should add tests for this change in text_editor. Are u sure that this changes are necessary exactly here?

Reply:
https://github.com/DevExpress/DevExtreme/pull/22772/commits/f81dc1c6a5a31f5b9a0bb82c18ba0b68db2016e5
https://github.com/DevExpress/DevExtreme/pull/22772/commits/aca844e4b81e0c308de49409c6b672d9a57ccfee